libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Generators/LanguageGeneratorManager.cs

```diff
@@ -70,7 +70,7 @@ public static ImportResolverDelegate ResourceExplorerResolver(string locale, Dic
                 else
                 {
                     var content = resource.ReadTextAsync().GetAwaiter().GetResult();
```

Comment:
>.ReadTextAsync().GetAwaiter().GetResult(); [](start = 42, length = 42)

eeesh

Reply:
is there a better way to read async data and feed into a sync method?